src/mod_learnspells.cpp

```diff
@@ -66,7 +62,15 @@ class LearnSpellsOnLevelUp : public PlayerScript
         }
     }
 
-    void OnLevelChanged(Player* player, uint8 oldLevel)
+    void OnFirstLogin(Player *player) override
```

Comment:
I think we attack the * to its type instead

Reply:
This change was actually made by my auto formatter. I think its better to stick to it as I assume that it implements the current code style c++ guidelines.  

Additionally greping thru the Core's `Player.cpp` I found no usages of the `Thing* t` style but plenty of `Thing *t` style.

Reply:
lol and i thought only me was wondering that :P

Reply:
> Additionally greping thru the Core's Player.cpp I found no usages of the Thing* t style but plenty of Thing *t style.

I dont know what you see, but I only see `Type* t`

Reply:
![grafik](https://user-images.githubusercontent.com/876676/77707252-e2f1ec00-6fc4-11ea-984a-960cf5d495f9.png)
![grafik](https://user-images.githubusercontent.com/876676/77707259-e84f3680-6fc4-11ea-8e2f-8fe3efa0000b.png)
![grafik](https://user-images.githubusercontent.com/876676/77707288-fdc46080-6fc4-11ea-81d6-576b7609a0ca.png)
![grafik](https://user-images.githubusercontent.com/876676/77707295-05840500-6fc5-11ea-8445-c7eab8a02b42.png)
![grafik](https://user-images.githubusercontent.com/876676/77707328-1af92f00-6fc5-11ea-9a93-fd373981aabc.png)
![grafik](https://user-images.githubusercontent.com/876676/77707337-2187a680-6fc5-11ea-9ba6-5c361f02b654.png)
![grafik](https://user-images.githubusercontent.com/876676/77707367-35cba380-6fc5-11ea-8937-d9dc536d973d.png)
![grafik](https://user-images.githubusercontent.com/876676/77707391-4845dd00-6fc5-11ea-8e66-219701a4880e.png)

Reply:
![grafik](https://user-images.githubusercontent.com/876676/77707452-73303100-6fc5-11ea-8dca-7a1f1cbbcd23.png)


source: https://geosoft.no/development/cppstyle.html#Types

Reply:
Just for clarification, I dont want to be nit-picking too much. 
Leave it as is, if you want.

Reply:
Ok I see. I've already reformatted Player.cpp on my version. That's where the confused comes from. Anyhow I do prefer the `Thing*` style. If I find a clag formatter option I will reformat.